build/scripts/entrypoint-volume.sh

```diff
@@ -29,4 +29,4 @@ else
 fi
 
 # Launch che with a custom connection-token
-./node out/vs/server/main.js --port 3100 --connection-token eclipse-che
+./node out/vs/server/main.js --port 3100 --connection-token eclipse-che --folder ${CHE_PROJECTS_ROOT}
```

Comment:
AFAIK CHE_PROJECTS_ROOT is only available for che-server workspaces

Reply:
You're right. It's `PROJECTS_ROOT` for the DevWorkspace engine. I forgot about that.
I'm updating the PR...
Thanks for noticing it!

Reply:
Also I think it should open the folders defined by the projects of the devfile.yaml
Here it might open the root folder which may not what we expect.
I'm fine to merge this one as it's still better than before but probably we need to do the same stuff than for che-theia where we create a .workspace file and open that .workspace file

Now it's quite easy as projects are cloned before we open the project

Reply:
As I remember, Che-Code editor is supposed to work with DevWorkspace engine only. So, I've replaced `CHE_PROJECTS_ROOT` with `PROJECTS_ROOT`.

Reply:
> Also I think it should open the folders defined by the projects of the devfile.yaml

We can add the custom folders support as one of further improvements.